sdks/python/apache_beam/runners/portability/portable_runner_test.py

```diff
@@ -82,19 +82,12 @@ def _pick_unused_port():
 
   @classmethod
   def _start_local_runner_subprocess_job_service(cls):
-    if cls._subprocess:
-      # Kill the old one if it exists.
-      cls._subprocess.kill()
+    cls._maybe_kill_subprocess()
     # TODO(robertwb): Consider letting the subprocess pick one and
     # communicate it back...
     port = cls._pick_unused_port()
     logging.info('Starting server on port %d.', port)
-    cls._subprocess = subprocess.Popen([
-        sys.executable, '-m',
-        'apache_beam.runners.portability.local_job_service_main', '-p',
-        str(port), '--worker_command_line',
-        '%s -m apache_beam.runners.worker.sdk_worker_main' % sys.executable
-    ])
+    cls._subprocess = subprocess.Popen(cls._subprocess_command(port))
```

Comment:
This will stop us from running multiple tests in parallel.
Does python run multiple tests in parallel?